core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -667,7 +667,15 @@ public MetricKey build() {
           .setDescription("The last raft log index which was applied to the state machine")
           .setMetricType(MetricType.GAUGE)
           .build();
-
+  public static final MetricKey MASTER_JOURNAL_CHECKPOINT_WARN =
+      new Builder("Master.JournalCheckpointWarn")
+          .setDescription(String.format("If the raft log index exceeds %s, and the "
+              + "last checkpoint exceeds %s, it returns 1 to indicate that a warning"
```

Comment:
@yuzhu what's your opinion in this? 1:0 or true:false.
Also what's normal in the metrics world?

Reply:
I personally prefer to use boolean and i do saw other java metrics expose as boolean

Reply:
If some Java project metrics use boolean, I think it can be used in this way.